--- tist_simple_button.js	2015-04-08 15:41:44.423392607 +0900
+++ gyro.js	2015-04-08 15:41:23.187392987 +0900
@@ -28,6 +28,21 @@
 	});
 }
 
+function ti_gyroscope(conned_obj) {
+	var period = 1000; // ms
+	conned_obj.enableGyroscope(function() {
+		conned_obj.setGyroscopePeriod(period, function() {
+			conned_obj.notifyGyroscope(function() {
+				console.info("ready: notifyGyroscope");
+				console.info("notify period = " + period + "ms");
+				conned_obj.on('gyroscopeChange', function(x, y, z) {
+					console.log(x, y, z);
+				});
+			});
+		});
+	});
+}
+
 var SensorTag = require('sensortag');
 console.info(">> STOP: Ctrl+C or SensorTag power off");
 console.info("start");
@@ -38,6 +53,7 @@
 		sensorTag.readDeviceName(function(error, deviceName) {
 			console.info("connect: " + deviceName);
 			ti_simple_key(sensorTag);
+			ti_gyroscope(sensorTag);
 		});
 	});
 	/* In case of SensorTag PowerOff or out of range when fired `onDisconnect` */
